lib/ReactViews/DataCatalogGroup.jsx

```diff
@@ -1,64 +1,49 @@
 'use strict';
 
-import DataCatalogItem from './DataCatalogItem.jsx';
+import DataCatalogMember from './DataCatalogMember.jsx';
 import defined from 'terriajs-cesium/Source/Core/defined';
 import Loader from './Loader.jsx';
 import ObserveModelMixin from './ObserveModelMixin';
 import React from 'react';
+import classNames from 'classnames';
 
 const DataCatalogGroup = React.createClass({
     mixins: [ObserveModelMixin],
 
     propTypes: {
-        group: React.PropTypes.object,
-        previewedCatalogItem: React.PropTypes.object,
-        onPreviewedCatalogItemChanged: React.PropTypes.func,
-        isOpen: React.PropTypes.bool,
-        onToggleOpen: React.PropTypes.func
+        group: React.PropTypes.object.isRequired,
+        viewState: React.PropTypes.object.isRequired
     },
 
-    toggleOpen(e) {
-        if (defined(this.props.onToggleOpen)) {
```

Comment:
This strange-looking logic is needed for catalog item search.  When a group matches the search query, that group should be shown initially closed even if it was open in the catalog view.  So it can't use the normal `group.isOpen` property.